src/Microsoft.Health.Dicom.Core.UnitTests/Features/Retrieve/RetrieveRenderedServiceTests.cs

```diff
@@ -39,23 +39,28 @@ public class RetrieveRenderedServiceTests
     private readonly IDicomRequestContextAccessor _dicomRequestContextAccessor;
     private readonly RecyclableMemoryStreamManager _recyclableMemoryStreamManager;
     private readonly ILogger<RetrieveRenderedService> _logger;
-
+    private readonly ILoggerProvider _xunitLoggerProvider;
     private readonly string _studyInstanceUid = TestUidGenerator.Generate();
     private readonly string _firstSeriesInstanceUid = TestUidGenerator.Generate();
     private readonly string _secondSeriesInstanceUid = TestUidGenerator.Generate();
     private readonly string _sopInstanceUid = TestUidGenerator.Generate();
     private static readonly CancellationToken DefaultCancellationToken = new CancellationTokenSource().Token;
 
 
-    public RetrieveRenderedServiceTests()
+    public RetrieveRenderedServiceTests(ITestOutputHelper output)
```

Comment:
It could also behoove us to encapsulate this in a test fixture that exposes an `ILoggerFactory`?